content/source/docs/enterprise/private/faq.html.md

```diff
@@ -152,6 +153,22 @@ You will again be asked if you want to copy the state file. Type `yes` and the s
 
 ---
 
+## Outbound Access
+
+This is a list of hostnames that the product connects to and should be available for egress:
+
+* releases.hashicorp.com
```

Comment:
We probably should indicate that this is optional if you're willing to supply a bundle, with a link to `/docs/enterprise/run/index.html#custom-and-community-providers`.